watertap/unit_models/translator_adm1_asm1.py

```diff
@@ -0,0 +1,463 @@
+            doc="""Property parameter object used to define property
+calculations for the outgoing stream,
+**default** - None.
+**Valid values:** {
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "outlet_property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property package "
+            "of the outgoing stream",
+            doc="""A ConfigBlock with arguments to be passed to the property
+block associated with the outgoing stream,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}""",
+        ),
+    )
+    CONFIG.declare(
+        "reaction_package",
+        ConfigValue(
+            default=None,
+            domain=is_reaction_parameter_block,
+            description="Reaction package to use for control volume",
+            doc="""Reaction parameter object used to define reaction calculations,
+**default** - None.
+**Valid values:** {
+**None** - no reaction package,
+**ReactionParameterBlock** - a ReactionParameterBlock object.}""",
+        ),
+    )
+    CONFIG.declare(
+        "reaction_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing reaction packages",
+            doc="""A ConfigBlock with arguments to be passed to a reaction block(s)
+and used when constructing these,
+**default** - None.
+**Valid values:** {
+see reaction package for documentation.}""",
+        ),
+    )
+
+    def build(self):
+        """
+        Begin building model.
+        Args:
+            None
+        Returns:
+            None
+        """
+        # Call UnitModel.build to setup dynamics
+        super(TranslatorData, self).build()
+
+        self.i_ec = Param(
+            initialize=0.06,
+            units=pyunits.dimensionless,
+            mutable=True,
+            doc="Nitrogen inert content",
+        )
+
+        # Add State Blocks
+        self.properties_in = self.config.inlet_property_package.build_state_block(
+            self.flowsheet().time,
+            doc="Material properties in incoming stream",
+            defined_state=True,
+            has_phase_equilibrium=False,
+            **self.config.inlet_property_package_args
+        )
+
+        self.properties_out = self.config.outlet_property_package.build_state_block(
+            self.flowsheet().time,
+            doc="Material properties in outgoing stream",
+            defined_state=self.config.outlet_state_defined,
+            has_phase_equilibrium=False,
+            **self.config.outlet_property_package_args
+        )
+
+        # Add ports
+        self.add_port(name="inlet", block=self.properties_in, doc="Inlet Port")
+        self.add_port(name="outlet", block=self.properties_out, doc="Outlet Port")
+
+        mw_n = 14.006 * pyunits.kg / pyunits.kmol
+        mw_c = 12.011 * pyunits.kg / pyunits.kmol
```

Comment:
Do we use these numbers in the corresponding property/reaction models for ASM1/ADM1? I only ask because I seem to recall us using something like 14 instead of 14.006 and 12 instead of 12.011 (I could be wrong.)

Maybe it isn't critical, but it could lead to inconsistencies that we'd want to avoid.